### DISCUSSION TITLE
Creating a Retro Mini Cassette Player with Raspberry Pi PICO by micropython

### DISCUSSION THREAD

**Discussion by jd3096-mpy** [micropython/micropython#13352] [Ideas]
Hello everyone, I'm jd3096 and I started learning programming at the age of thirty. As a self-taught MicroPython learner for two and a half years, this is my first attempt to make a project by the Raspberry Pi PICO. Thank you for your supporting. I enjoy trying projects that others haven't done.
![微信图片_20240102103640](https://github.com/micropython/micropython/assets/69076780/d1fe2aa5-a5b9-4524-95b3-626b4f355ab2)
See here:
https://community.dfrobot.com/makelog-313900.html

**Comment by sophiedegran** [comment]
Very nice project! 
It's amazing what you've done!
